jango2106/traveller
# App to flask

# Issues #5 

## Updates
Added a hello world flask app to the project
Dockerized the app/api
Updated readme docs

## How to test:
1. Follow directions in readme to start the api
2. In a browser go to `localhost:5000` to see a hello world message
